### PR TITLE
Add LICENSE, PATENTS and CONTRIBUTING.md files.

### PR DESCRIPTION
**Pull request by rrh**
The text is copied directly from oss-performance, with substitutions
to use the term "HHVM packaging".

This addresses https://github.com/hhvm/packaging/issues/106

This is a little creepy.  My developer license agreement lets me make
contributions, but I'm in here creating legal boilerplate describing
how I can make contributions.

### CODE REVIEW

**Comment by jwatzman** [comment]
Thanks for the PR!

> This is a little creepy. My developer license agreement lets me make
> contributions, but I'm in here creating legal boilerplate describing
> how I can make contributions.

Haha, don't worry, we'll run this stuff by legal on our end before doing anything. (Which I'm gonna go do now.)

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Please note that this repository includes archives of the software being
+benchmarked, which is licensed separately, under different terms; see the
+notices inside the archives for details.
```
Comment (fredemmott) [inline LICENSE]: copypasta :)

**Comment by rrh** [comment]
Well, the entire packing git repo contains a bunch of embedded .so files, most of which don't seem to be from facebook, and are compiled from source from a bunch of different licenses.

I don't care for these embedded .so's anyway, since it makes the github download be ~100Mb, but I don't understand enough of the history and architecture to really comment as to their usefullness to support the down-rev platforms your customers run this on.

**Comment by rrh** [comment]
I made a stab at fixing the cut&paste'o from oss-performance, but because of the plethora of .so files, their potentially varying terms (which I haven't chased down), issues with licensing binary code, etc, you'll certainly need to discuss this with somebody who is indeed a lawyer.

**Comment by rrh** [comment]
Background:  I'd like to use this packaging software to build a deb to ship our build of hhvm.  I expect that my handful of changes will incrementally get worked into the main hhvm in various PRs over the next months, but not yet.

I can't use any 3rd party software unless there's a license that our officers or lawyers are happy with.

**Comment by jwatzman** [comment]
Going to temporarily hold off on this PR (though we eventually plan to merge it).

**Comment by rrh** [comment]
Roughly how long is "eventually"?

**Comment by jwatzman** [comment]
Something fairly small we need to do first, but not hard, just a matter of someone getting around to it.

(Sorry to be so obnoxiously vague. Never sure how much I am allowed to commit to in writing about legal stuff.)

**Comment by jwatzman** [comment]
Went ahead and added copies manually since the verbiage about the packaged `.so` files no longer applies.

**Pull request closed without merging**